### PR TITLE
FCCEUD-36 Product versioning

### PR DESCRIPTION
**Pull request by stoobie**
@redhataccess/pantheon-developers Please review for technical completeness and accuracy. Please ignore the files con_about_products.adoc and proc_creating-a-product.adoc. I mistakenly created this branch from FCCEUD-35 instead of from master.

### CODE REVIEW

**Comment by adahms** [comment]
@redhataccess/pantheon-developers - is anyone available to do a tech review on this one?

**Comment by anarang** [comment]
@adahms @stoobie looks good to me! tech review complete.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_versioning_a_product.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+.Prerequisites
+
+* You are logged in.
+* The product to be versioned exists in {ProductName}.
+
+.Procedure
+
+. Navigate to menu:Products[Product Listing].
+
+. Find the product and click the dropdown arrow>Product Details.
```
Comment (RichardHoch) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_versioning_a_product.adoc:15]: drop-down

Comment (stoobie) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_versioning_a_product.adoc:15]: Fixed.

**Review by RichardHoch — APPROVED** [review]

**Comment by adahms** [comment]
@stoobie - ready to merge?

**Review by RichardHoch — APPROVED** [review]

**Comment by RichardHoch** [comment]
I approved the changes (again) just now. Not sure why this is blocked but it has passed (my) peer review.

**Comment by adahms** [comment]
@redhataccess/pantheon-developers - ready for merge.

**Review by benradey — APPROVED** [review]

**Pull request merged**